src/Illuminate/Queue/Worker.php

```diff
@@ -218,8 +241,6 @@ public function runNextJob($connectionName, $queue, WorkerOptions $options)
         if ($job) {
             return $this->runJob($job, $connectionName, $options);
         }
-
-        $this->sleep($options->sleep);
```

Comment:
You are right - haven't noticed that listener executes "queue:work" with "once" and "sleep". Having said that, I can see that "queue:listen" is not even in the Queues documentation (https://laravel.com/docs/5.4/queues) anymore? Is it about to get deprecated?